content/install-guides/armpl.md

```diff
@@ -36,84 +36,73 @@ Arm Performance Libraries are available for use on [Windows 11 on Arm](#windows)
-- Linux: &emsp;&emsp;&emsp;&ensp;&nbsp;&ensp;&ensp;&nbsp;[Get started with Arm Performance Libraries (stand-alone Linux version)](https://developer.arm.com/documentation/102620/2310/?lang=en)
+- Windows: &emsp;&emsp;&emsp;&nbsp;[Get started with Arm Performance Libraries (Windows version)](https://developer.arm.com/documentation/109361/latest)
+- MacOS: &emsp;&emsp;&emsp;&ensp;&nbsp;&nbsp;&nbsp;[Get started with Arm Performance Libraries (macOS version)](https://developer.arm.com/documentation/109362/latest)
+- Linux: &emsp;&emsp;&emsp;&ensp;&nbsp;&ensp;&ensp;&nbsp;[Get started with Arm Performance Libraries (stand-alone Linux version)](https://developer.arm.com/documentation/102620/latest)
 
 ## Windows {#windows}
 
-On your Windows 11 Arm machine, go to the [Arm Performance Libraries download page](https://developer.arm.com/downloads/-/arm-performance-libraries). Click on the Download Windows section. You will be prompted to review and accept the End User License Agreement before you can download the zip file. Click on the `I accept the terms of this License Agreement` checkbox and proceed to `Download` as shown below.
+On your Windows 11 Arm machine, go to the [Arm Performance Libraries download page](https://developer.arm.com/downloads/-/arm-performance-libraries). Click on the Download Windows section. You will be prompted to review and accept the End User License Agreement before you can download the zip file. Click on the `I accept the terms of this License Agreement` checkbox and proceed to `Download`.
 
-![win_download #center](/install-guides/_images/download-win-armpl_23.10.png)
-
-Open your Windows File Explorer and locate the downloaded `arm-performance-libraries_23.10.zip` file.
+Open your Windows File Explorer and locate the downloaded `arm-performance-libraries_24.04.zip` file.
 Extract the contents of this zip file using the "Extract all" button at the top of the Windows File Explorer.
 
-### Update your system environment variables
-
-Using Windows Search, open `Edit the System Variables` in the `Control Panel`.
-On the `Advanced Tab` of the `System Properties` window, click on the `Environment Variables` button.
-
-![sys_prop #center](/install-guides/_images/windows-sys-prop.png)
-
-Add a New variable called `ARMPL_DIR` which should point to the location where you unpacked the Arm Performance Libraries.
-
-![add_var #center](/install-guides/_images/windows-sys-env_23.10.png)
-
-Edit the `Path` variable to add `%ARMPL_DIR%\bin` to the list of existing directories in your path.
+### Install
 
-![edit_path #center](/install-guides/_images/win-sys-path.png)
+Installation instructions for ArmPL on Windows can be found below:
 
-You can now start linking your application to the Arm Performance libraries on your Windows on Arm device. Follow the examples in the included `RELEASE_NOTES` file of your extracted installation directory to get started.
+https://developer.arm.com/documentation/109408/0100/Installation/Install-on-Windows
```

Comment:
Could you write the title of the document you are referencing and then use a link to the referenced document?